packages/preview/iconic-salmon-svg/2.1.0/iconic-salmon-svg.typ

```diff
@@ -0,0 +1,268 @@
+#import "./iconic-salmon-svg-icons.typ" as icon
+
+/// A function that creates a clickable link with the passed name as text
+/// In addition to this, the associated icon of the web page is inserted
+///
+/// - name (str): The visible name of the clickable link
+/// - url (str): The URL to the website (it is assumed to start with the http/https scheme)
+/// - icon_provider (function): The function that provides the associated icon
+/// -> content
+#let icon-link-generator(
+  name,
+  url,
+  icon_provider,
```

Comment:
I would suggest to rename this parameter (here and in all other places you use it) to `icon-provider` to be more consistent with other parts of your package and more generally the Typst ecosystem.

Reply:
Indeed, thanks for the hint! :smiley:

Reply:
Fixed. I also changed `icon_func` to `icon-func`.